tests/rptest/tests/full_node_recovery_test.py

```diff
@@ -34,6 +34,7 @@ def __init__(self, test_context):
         super(FullNodeRecoveryTest, self).__init__(test_context=test_context,
                                                    extra_rp_conf=extra_rp_conf)
 
+    @ignore  # Temporarily disabled: see issue #3880
```

Comment:
```suggestion
    @ignore  # https://github.com/redpanda-data/redpanda/issues/3880
```

nitpick: use full url so easier to navigate to the issue

Reply:
Thanks for the review. I prefer the existing version here--making clear it is temporary and keeping on a single line. We reference issues so often I prefer the short form, and my editor does allow click-to-navigate:
![gh-integration](https://user-images.githubusercontent.com/558234/155411605-30fe6105-1d8f-4111-9753-b7e592fbcbba.png)